test/verify/check-sosreport

```diff
@@ -52,9 +52,8 @@ only-plugins=release,date,host,cgroups,networking
         b.click("button:contains('Create report')")
         b.wait_visible("#sos-dialog")
 
-        with b.wait_timeout(540):
```

Comment:
Note that as long as everything is as expected, this will *not* wait for 9 minutes regularly. It will wait *up to* 9 minutes, just like a normal b.wait_* will wait *up to* 60s, but of course immediately return once the condition gets true.

This is a correct change -- we expect to write a config which is fast, and apparently we violate this somewhere. So let's find out where/how. Thanks!

Reply:
> Note that as long as everything is as expected, this will _not_ wait for 9 minutes regularly. It will wait _up to_ 9 minutes, just like a normal b.wait_* will wait _up to_ 60s, but of course immediately return once the condition gets true.

Yes, that's what I mentioned in the commit on Fedora-36 sos report is broken so we wait 9 minutes (2x)

> This is a correct change -- we expect to write a config which is fast, and apparently we violate this somewhere. So let's find out where/how. Thanks!

I am not following what you mean here? Writing the config file was not fast or made it slower?

Reply:
I meant that we *may* write the wrong config contents, something that the current version of sos can't understand any more. sosreport taking very long (minutes instead of seconds) would be a symptom of that. This already happened for sos 3 → 4.

Reply:
Ok, is there a proposal to do it different, we could use sed instead but that feels a lot more brittle as not all configuration options are available or commented out by default:

```
[root@rhel-8-6-127-0-0-2-2201 ~]# grep threads /etc/sos/sos.conf
[root@rhel-8-6-127-0-0-2-2201 ~]# grep only-plugins /etc/sos/sos.conf
[root@rhel-8-6-127-0-0-2-2201 ~]#
```